test/test_client_db_hooks.js

```diff
@@ -0,0 +1,63 @@
+import db         from '../src/models/db';
+import * as CONST from './constants';
+import { hookInvoked, resetHooks } from './server';
+
+db().then(models => {
+  describe('Database hooks', () => {
+    beforeEach(() => {
+      resetHooks();
+      return models.sequelize.transaction(transaction => {
+        return Promise.all(Object.keys(CONST.entities).map(name => {
+          return models[name].destroy({ transaction, where: {} });
+        }));
+      });
+    });
+
+    Object.keys(CONST.entities).forEach(entity => {
+      it(entity + ' "create" hooks', done => {
+        models[entity].create(CONST[entity + 'Entity']).then(() => {
+          CONST.dbHooks.slice(0, 2).forEach(hook => {
```

Comment:
I'm going to move the `dbHooks` from `src/constants.js` (where they are not needed anymore based on https://github.com/mozilla-sensorweb/sensorthings/pull/235#discussion_r98526959) into `test_client_db_hooks.js`.